native/dll/DX93D/SageTVDX93D.cpp

```diff
@@ -3092,7 +3105,7 @@ JNIEXPORT jboolean JNICALL Java_sage_DirectX9SageRenderer_getVideoSnapshot0
 	{
 // NOTE: WE SHOULD FIX THIS SO ITS NOT A PROBLEM
 		pSnapshotSurf->UnlockRect();
-		hr = -1;
+		hr = E_FAIL;

```

Comment:
Why are you changing this?

Reply:
Static Analysis gives warning
`warning C6226: Implicit cast between semantically different integer types:  assigning -1 to HRESULT.  Consider using E_FAIL instead.`